src/FontManager.cpp

```diff
@@ -1,5 +1,5 @@
 /***************************************************************************
- *   Copyright (C) 2009-2014 by Vadim Peretokin - vperetokin@gmail.com     *
+ *   Copyright (C) 2009-2018 by Vadim Peretokin - vperetokin@gmail.com     *
```

Comment:
Have you *really* modified **this** file every single year since the project started? 😉

Reply:
Adjusted.